docs/general/passing_arbitrary_options.md

```diff
@@ -0,0 +1,45 @@
+## Passing Arbitrary Options To A Serializer
+
+Let's say you have a basic Post Controller:
+
+```ruby
+class PostController < ApplicationController
+  def dashboard  
+    render json: @posts
+  end
+end
+```
+
+Odds are, your serializer will look something like this:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body
+end
+```
+
+This works all fine and well, but maybe you passing in some "arbitrary" options
+into the serializer. Here's what you would do:
+
+### posts_controller.rb
```

Comment:
I don't know what's your opinion on this but I usually find easier to easier to understand when having the full code snippet instead of just part of it. Also I usually tend to prefer having one snippet with comments to specify the filename instead of using h3 here. An example here would be:

```ruby
# post_controller.rb
class PostController < ApplicationController
 def dashboard  
   render json: @posts, user_id: 12
 end
end

# post_serializer.rb
class PostSerializer < ActiveModel::Serializer
  attributes :id, :title, :body
  has_many :comments_by_me
  def comments_by_me  
    Comments.where(user_id: instance_options[:user_id], post_id: object.id)
  end
end
```

Maybe that's just my opinion and might not be true for everyone so let me know what you guys think.